### ISSUE TITLE
installing snaps from snapcraft

### ISSUE THREAD

**Issue by SandroCristiano** [bartzaalberg/snaptastic#68]
Whenever I try to install from snapcraft.io, snaptastic opens, shows me the install dialog and after I click install nothing happens for a while and then snaptastic closes and the snap isn't installed. It works the other way around, though. If I install a snap using the command line, it installs normally and is shown in the snaptastic window. 

Just a heads up: I've installed a snap via command line and after that all others were able to be installed via snaptastic.

**Comment by unbelievableflavour** [comment]
Do you know which Snap it was? I can't really debug it if i don't know where it went wrong :)

**Comment by SandroCristiano** [comment]
It happen when I tried to install Discord.

**Comment by unbelievableflavour** [comment]
I just installed discord and it was fine. probably something to do with first snap ever installing.

**Comment by SandroCristiano** [comment]
Yes. Discord was just an example. I reinstalled elementary onw of these
days and still had the same problem: snaptastic only install snaps after a
snap is first installed via snapd in the terminal.

**Comment by sysfu** [comment]
I've already successfully installed several snaps via the command line. I installed Snaptastic the other day then visited https://snapcraft.io/vlc

After clicking the green install button on the upper right, then the "View in Desktop store" button, I get the following dialogues

![Screenshot from 2019-04-23 07-54-12](https://user-images.githubusercontent.com/829699/56591558-d3407b00-659d-11e9-8b37-a5c180719048.png)


![Screenshot from 2019-04-23 07-51-50](https://user-images.githubusercontent.com/829699/56590949-c707ee00-659c-11e9-9efd-f65aeadd59e1.png)

I click "Open xdg-open" for the first and then "Yes" on the second and then nothing happens. This occurs with the Epiphany, Brave, and Vivaldi browsers.

**Comment by peteruithoven** [comment]
Have you tried: https://github.com/bartzaalberg/snaptastic#snaptastic-wont-handle-snap-urls

**Comment by sysfu** [comment]
I'm not using Firefox so I am not sure if the suggested workaround applies in my case.

Also I cannot find any comparable settings in Chromium browser engine settings for either of the three browsers I mentioned.

**Comment by sysfu** [comment]
I also tried running `sudo update-desktop-database /usr/share/applications` and that did not resolve either.

**Comment by unbelievableflavour** [comment]
Ok, have you tried other browsers than Epiphany, Brave, or Vivaldi?

**Comment by sysfu** [comment]
I tried Firefox and it works. Chromium tries to open snaps such as https://snapcraft.io/riot-web in the Gnome 'Software' app, instead of snaptastic or Elementary's AppCenter

**Comment by o-alquimista** [comment]
Same as @sysfu here. It prompts to install Canonical's official Snap Store instead of using Snaptastic. I'm using Epiphany browser on a barely modified elementary Juno install.
